test/unit/cpu/test-forall.cxx

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
I'll add an issue to take a pass through the code and make the copyright/license reference comments the same in all files...